website/src/components/User/GroupPage.tsx

```diff
@@ -94,22 +95,32 @@ const InnerGroupPage: FC<GroupPageProps> = ({
                         </div>
                     </h1>
                     {userIsGroupMember && (
-                        <button
-                            onClick={() => {
-                                displayConfirmationDialog({
-                                    dialogText: `Are you sure you want to leave the ${groupName} group?`,
-
-                                    onConfirmation: async () => {
-                                        await removeFromGroup(username);
-                                        window.location.href = routes.userOverviewPage();
-                                    },
-                                });
-                            }}
-                            className='object-right p-2 loculusColor text-white rounded px-4'
-                            disabled={!isClient}
-                        >
-                            Leave group
-                        </button>
+                        <>
+                            <button
+                                className='object-right p-2 loculusColor text-white rounded px-4 mr-2'
+                                onClick={() => {
+                                    window.location.href = routes.editGroupPage(groupId);
+                                }}
+                            >
+                                Edit group
+                            </button>
```

Comment:
Why don't we make this an `<a>` tag? It's essentially only a link. Browsers don't need JS for that.

Reply:
good point, I had just copied this way of doing it from the button below which also has a confirmation dialog.

I've changed it now: https://github.com/loculus-project/loculus/pull/2854/commits/2f2ce7eadf1c703a1946e17de5f19be15501be6e